src/HealthCheckServer.ts

```diff
@@ -22,9 +22,8 @@
  --------------
  ******/
 
-import Hapi, { Lifecycle } from '@hapi/hapi';
-import Boom from '@hapi/boom';
-
+import Hapi, { Lifecycle, ResponseObject } from '@hapi/hapi'
+import Boom from '@hapi/boom'
```

Comment:
I don’t think boom is needed in any of our api’s anymore as we have our own error handling framework